test/uk/gov/hmrc/directdebitupdateemailbackend/controllers/UpdateSelectedEmailControllerSpec.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.directdebitupdateemailbackend.controllers
+
+import ddUpdateEmail.connectors.JourneyConnector
+import ddUpdateEmail.models.Email
+import play.api.test.Helpers._
+import uk.gov.hmrc.crypto.Sensitive.SensitiveString
+import uk.gov.hmrc.directdebitupdateemailbackend.testsupport.stubs.AuthStub
+import uk.gov.hmrc.directdebitupdateemailbackend.testsupport.{ItSpec, TestData}
+import uk.gov.hmrc.http.{HeaderCarrier, UpstreamErrorResponse}
+
+class UpdateSelectedEmailControllerSpec extends ItSpec {
+
+  lazy val journeyConnector = app.injector.instanceOf[JourneyConnector]
+
+  "Requests to update the selected email" - {
```

Comment:
maybe chuck the route in the test name? not fussed though, since there is only one route in the controller. Feel free to ignore

Reply:
I thought about doing that and reached towards the reverse routing stuff to get a realiable url in there but there was the journey id  in the url to contend with there so I just went with this. With one thing per controller it's unlikely to cause confusion I hope

Reply:
changed my mind - it does look a bit better with the url in there. Just changed it